src/index.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 /**
  * @module braintree-web-drop-in
- * @description The primary way to integrate Drop-in into your page is to use [`dropin.create`](#.create).
+ * @description There are two ways to integrate Drop-in into your page: a script tag integration and a JavaScript integration using [`dropin.create`](#.create).
  *
- * Alternatively, if you only need to process credit cards on your checkout page, the script tag integration is the simplest way to integrate. All you need to do is add the Drop-in script inside your form element where you want Drop-in to appear and include a `data-braintree-dropin-authorization` property with your [tokenization key](https://developers.braintreepayments.com/guides/authorization/tokenization-key/javascript/v3) or [client token](https://developers.braintreepayments.com/guides/authorization/client-token).
+ * The script tag integration is the fastest way to integrate. All you need to do is add the Drop-in script inside your form element where you want Drop-in to appear and include a `data-braintree-dropin-authorization` property with your [tokenization key](https://developers.braintreepayments.com/guides/authorization/tokenization-key/javascript/v3) or [client token](https://developers.braintreepayments.com/guides/authorization/client-token).
  *
- * The script tag integration will intercept the form submission and attempt to tokenize the credit card. If the tokenization is successful, it will insert the payment method nonce representing the credit card into a hidden input with the name `payment_method_nonce` and then submit your form.
+ * When your form is submitted, Drop-in will intercept the form submission and attempt to tokenize the payment method. If the tokenization is successful, it will insert the payment method nonce into a hidden input with the name `payment_method_nonce` and then submit your form. If the tokenization is unsuccessful, a relevant error will be shown in the UI.
+ *
+ * Specify creation options as data attributes in your script tag, as shown in the examples below. The following configuration properties may be set: `locale`, `payment-option-priority`, `paypal.amount`, `paypal.currency`, `paypal.flow`, `paypal-credit.amount`, `paypal-credit.currency`, `paypal-credit.flow`.
```

Comment:
I'd be tempted to have a table with the full data attribute and a description unless we think that's redundant with the `create` options.